src/frontend/js/index.js

```diff
@@ -162,6 +162,8 @@ function zones2summary(summariser, state) {
         (state.percent && state.compare ? "" : " ") + getUnit(state.meta, "od_matrices", state.layers.od_matrices.variable, state.compare && state.percent)
 }
 
+// convert a weight between 0 and 1 to a hsl value between (230, 20%, 32%) and (283, 100%, 32%)
+const weightToColor = weight => `hsl(${230 + weight * 53}, ${20 + weight * 80}%, 32%)`
```

Comment:
I think a slightly different name would make clear that it's to do with the centroid lines, or better, make it a closure inside paintCentroids.

We have a few other places where we want to convert a float to a colour.